release-process.md

```diff
@@ -113,7 +110,7 @@ $ svn co https://dist.apache.org/repos/dist/dev/spark/
 mkdir spark-1.1.1-rc2
  
 # Download the voted binaries and add them to the directory
-$ scp andrewor14@people.apache.org:~/public_html/spark-1.1.1-rc2/* spark-1.1.1-rc2
+$ sftp -r andrewor14@people.apache.org:~/public_html/spark-1.1.1-rc2/* spark-1.1.1-rc2
```

Comment:
Just out of curiosity, is `sftp` required? or would `scp -r .../spark-1.1.1-rc2 .` work as well?

Reply:
Not anymore, they disabled scp access to people.apache and limited it to sftp only (super frustrating :( )